test/jdk/java/lang/management/ThreadMXBean/VirtualThreadDeadlocks.java

```diff
@@ -27,7 +27,8 @@
  * @summary Test ThredMXBean.findMonitorDeadlockedThreads with cycles of
  *   platform and virtual threads in deadlock
  * @enablePreview
- * @modules java.management
+ * @modules java.base/java.lang:+open java.management
```

Comment:
Can you explain the need for this change?

Reply:
That is jtreg foo to opens java.lang, need to access a private field.

Reply:
ok